include/util/coordinate_calculation.hpp

```diff
@@ -85,16 +85,6 @@ double bearing(const Coordinate first_coordinate, const Coordinate second_coordi
 // Get angle of line segment (A,C)->(C,B)
 double computeAngle(const Coordinate first, const Coordinate second, const Coordinate third);
 
-// find the center of a circle through three coordinates
```

Comment:
Added it back.

What's the advantage of having unused code over just getting it from git later if it's needed?